views/index.erb

```diff
@@ -144,6 +144,17 @@
                     Simulate CSP Error
                   </label>
                 </div>
+                <div class="usa-checkbox">
+                  <input class="usa-checkbox__input"
+                         id="ip-auth-option"
+                         type="checkbox"
+                         name="ip_auth_option"
+                         value="true"
```

Comment:
Optional: I think we could set the value here to the constant to be able to pass the value through directly.

```suggestion
                         value="mocked-auth-code-for-testing"
```

Directly as in:

```rb
      request_params = {
        // ...
        inherited_proofing_auth: params['ip_auth_option'],
      }.compact.to_query
```

Reply:
Andrew, I took your advice; seems like a good idea (moving values into the template). But, now, I can't figure out how to actually test the value, because I don't know how to set the checkbox, before the template is rendered (without using Capybara).  I have it testing, like it was before, ie. explicitly passing the param to the (controller) action:
`get '/', { ip_auth_option: va_test_auth_code }
`
But, this takes the checkbox value out of the test.

Reply:
Are you trying to avoid hard-coding the value in the test? I agree Capybara would be a nicer way to do this (test redirect of the submitted form), but maybe we could just render the page once without any parameters and pull the value from the HTML. Is that what you had in mind?

```rb
get '/'

doc = Nokogiri::HTML(last_response.body)

va_test_auth_code = doc.at('[name=ip_auth_option]').attr('value')

get '/', { ip_auth_option: va_test_auth_code }

doc = Nokogiri::HTML(last_response.body)
login_link = doc.at("a[href*='#{authorization_endpoint}']")
# ...
```

Reply:
Yes. And, yes. I couldn't see how to do that; I **was** thinking it was a double-call. Yeah, not the best/obvious, but it gets-er-done. Thank you.  I think this is a wrap!